src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+const arrOfEmployees = [...document.querySelectorAll('li')];
+
+const convertToNumber = value => +value.slice(1).split(',').join('');
+
+function sortList(list) {
+  return list.sort((a, b) => convertToNumber(b.dataset.salary)
+    - convertToNumber(a.dataset.salary));
+}
+
+listOfEmployees.append(...sortList(arrOfEmployees));
```

Comment:
It works, but there is no reason to call `sortList `like this, because sort is a mutable method and changes the initial array
```suggestion
function sortList(list) {
  return list.sort((a, b) => convertToNumber(b.dataset.salary)
    - convertToNumber(a.dataset.salary));
}

sortList(arrOfEmployees)

listOfEmployees.append(...arrOfEmployees);
```